javalib/src/main/scala/java/lang/Enum.scala

```diff
@@ -7,4 +7,9 @@ abstract class Enum[E <: Enum[E]] protected (_name: String, _ordinal: Int)
   def ordinal(): Int              = _ordinal
   override def toString(): String = _name
   final def compareTo(o: E): Int  = _ordinal.compareTo(o.ordinal)
+  // equals and hashCode copied from scala.js
```

Comment:
Given that implementation here is trivial, lets drop this comment.

Reply:
Removed the comment and rebased onto master